pyk/src/pyk/__main__.py

```diff
@@ -27,24 +28,25 @@ def main():
     # This change makes it so that in most cases, by default, pyk doesn't run out of stack space.
     sys.setrecursionlimit(10 ** 7)
 
-    commandLineArgs = create_argument_parser()
-    args = vars(commandLineArgs.parse_args())
+    cli_parser = create_argument_parser()
+    args = vars(cli_parser.parse_args())
 
     kompiled_dir = Path(args['kompiled-dir'])
 
-    if args['verbose']:
-        logging.basicConfig(level=logging.DEBUG, format=_LOG_FORMAT)
-    else:
+    if not args['verbose']:
         logging.basicConfig(level=logging.WARNING, format=_LOG_FORMAT)
+    elif args['verbose'] == 1:
+        logging.basicConfig(level=logging.INFO, format=_LOG_FORMAT)
+    elif args['verbose'] > 1:
+        logging.basicConfig(level=logging.DEBUG, format=_LOG_FORMAT)
 
     if args['command'] == 'print':
         printer = KPrint(kompiled_dir)
+        _LOGGER.info(f'Reading Kast from file: {args["term"].name}')
         term = KAst.from_json(args['term'].read())
         if type(term) is dict and 'term' in term:
             term = term['term']
-        if term == mlTop():
-            args['output_file'].write(printer.pretty_print(term))
-        else:
+        if term != mlTop():
```

Comment:
I don't know, but that also seems an unrelated question ot this refactoring. This refactoring preserves old behavior, but makes it so we only have one write statement to the `args['output']`. Is there a better way to check for `mlTop()`? I can use that instead, but anything beyond that should go in it's own issue (unrelated to this PR).

Reply:
OK, we should be checking the label of the KApply.

Reply:
Once again, not related to this issue at all. The question to ask yourself when reviewing code is: "is this an improvement over the current situation?" If so, then there is no need to block merging, you can make an issue with checklists or an immediate follow-up PR with things that aren't breakages. I will fix it here, but keep in mind that every time we bounce a PR back and forth, we are wasting at least an entire day, especially when we're in different timezones across the world. That only has to happen 3 times for a feature to be blocked from merging for _an entire week_, which is insane when I am working with deadlines for conferences which are on the order of weeks out. I need to deliver useful results in terms of web interface of prover by end of august when I have a demo, and would rather do it on merged code than telling people to check out specific branches.

Reply:
Also, see Chesterton's Fence: https://fs.blog/chestertons-fence/

Reply:
https://github.com/runtimeverification/k/issues/2773

Reply:
I've backed out this change as well. Hopefully we can agree in the future that when changes are strictly improvements, we don't need to block merge because they could be _a little bit better_. "perfect is the enemy of the good".

Reply:
Right, right. Sorry I didn't mean to imply we should change it immediately.

Reply:
Then don't request changes! Hit Approve instead!